### DISCUSSION TITLE
Github Copilot Duplicate Payment Issue

### DISCUSSION THREAD

**Discussion by suleymanbyzt** [community/community#126877] [Copilot Conversations]
### Select Topic Area

Question

### Body

I experienced a payment issue with Github Copilot. Two separate payments were made for the same Copilot account between May 29 at 3:17 PM (UTC+3) and June 1 at 8:31 PM (UTC+3). How can I request a refund for this repeated transaction, or can someone assist me? If you check your systems, you will see that two separate payments were processed for the same account within four days. Thank you in advance for your support.

![WhatsApp Image 2024-06-01 at 21 00 24](https://github.com/community/community/assets/58400764/f8413461-bbfb-4f7a-b994-4d520fade4c2)

**Comment by suleymanbyzt** [comment]
![image](https://github.com/community/community/assets/58400764/625351e9-0a22-4aa1-b765-d1f794388652)

I would appreciate your support on this matter.

**Comment by davevad93** [comment]
Hi @suleymanbyzt ,

Considering that your issue is billing related, I doubt that you'll find a solution here in the Community Discussions so I think you should contact support by opening a ticket [here](https://support.github.com/contact/account) and select `Billing, payments or receipts`.

**Comment by suleymanbyzt** [comment]
Thank you for your support.

**Reply by davevad93** [reply]
You're welcome, I hope you can solve it, good luck!
